lib/leanplum_api/version.rb

```diff
@@ -1,3 +1,3 @@
 module LeanplumApi
-  VERSION = '3.0.3'
+  VERSION = '3.0.4'
```

Comment:
if you change the signature of `track_multi`, you have to do a major version bump, which i'm not sure i endorse, honestly.

Reply:
i endorse the major version bump to get the `device_attributes` in the arguments, actually.  change this to `4.0.0`

also need information in `CHANGELOG.md`